spec/server/functional/classrooms.spec.coffee

```diff
@@ -386,14 +387,16 @@ describe 'POST /db/classroom/-/members', ->
     @student = yield utils.initUser()
     done()
 
-  it 'adds the signed in user to the classroom and any free courses and sets role to student', utils.wrap (done) ->
+  it 'adds the signed in user to the classroom and any free courses and sets role to student and unsubscribes', utils.wrap (done) ->
+    spyOn(subscriptions, 'unsubscribeUser').and.returnValue(Promise.resolve());
     yield utils.loginUser(@student)
     url = getURL("/db/classroom/anything-here/members")
     [res, body] = yield request.postAsync { uri: url, json: { code: @classroom.get('code') } }
     expect(res.statusCode).toBe(200)
     classroom = yield Classroom.findById(@classroom.id)
     expect(classroom.get('members').length).toBe(1)
     expect(classroom.get('members')?[0]?.equals(@student._id)).toBe(true)
+    expect(subscriptions.unsubscribeUser).toHaveBeenCalled()
```

Comment:
Probably, but I think I'll leave it as is.  `stripe.customers.cancelSubscription` is the first interesting call in `unsubscribeUser`, and if we want to test `unsubscribeUser` internals it would be to do directly instead via a classroom joins test.